Sources/PenguinStructures/Graphs/AdjacencyList.swift

```diff
@@ -85,12 +88,26 @@ extension AdjacencyList: MutableGraph {
 		return EdgeId(source: source, destination: destination, offset: offset)
 	}
 
-	public mutating func removeEdge(from u: VertexId, to v: VertexId) {
+    /// Removes the edge (u, v) from the graph.
+    ///
+    /// If there are parallel edges, it removes all edges.
+    ///
+    /// - Precondition: `u` and `v` are valid `VertexId`s from `self`.
+    /// - Throws: `GraphErrors.edgeNotFound` if no edges are found.
```

Comment:
```suggestion
    /// - Throws: `GraphErrors.edgeNotFound` if there is no edge from `u` to `v`.
```

We can argue about whether that's a good idea tomorrow ;-)